hw2/05-form.js

```diff
@@ -0,0 +1,29 @@
+// Add your code here
+document.querySelector('form').addEventListener('submit', handleSubmit);
+
+function handleSubmit(event) {
+  event.preventDefault();
+  const name = document.getElementById('name').value;
+  const email = document.getElementById('email').value;
+  const registrationStatus = document.getElementById('registration-status').value || 'N/A';
+  const programmingLanguages = document.getElementById('programming-languages').checked;
+  const operatingSystems = document.getElementById('operating-systems').checked;
+  const fullStack = document.getElementById('full-stack').checked;
+  const message = document.getElementById('message').value || 'N/A';
+
+  console.group('===== Form Submission =====');
+  console.log(`Name: ${name}`);
+  console.log(`Email: ${email}`);
+  console.log(`Registration Status: ${registrationStatus}`);
+  if (programmingLanguages || operatingSystems || fullStack) {
+    console.group('Courses Taken:');
```

Comment:
I love that you made this into a console group as well. Looks great!

Reply:
Thank you!